webpack.dev.config.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable */
+'use strict';
+
+const webpack = require('webpack');
+const webpackConfig = require('./webpack.base.config');
+
+webpackConfig.mode = 'development';
+
+let dhisConfig;
+
+function getConfig() {
+    const dhisConfigPath = process.env.DHIS2_HOME && `${process.env.DHIS2_HOME.trimRight('/')}/config`;
+    console.log(`using config.json from ${dhisConfigPath}`)
+    return require(dhisConfigPath);
+}
+
+
+try {
+    dhisConfig = getConfig();
+    console.log(dhisConfig)
+} catch (e) {
+    // Failed to load config file - use default config
+    console.log('\nWARNING! Failed to load DHIS config:' + e.message);
+    dhisConfig = {
+        baseUrl: 'http://localhost:8080/',
+        authorization: 'Basic YWRtaW46ZGlzdHJpY3Q=', // admin:district
+    };
+}
+
+webpackConfig.plugins = [
+    new webpack.DefinePlugin({
+        DHIS_CONFIG: JSON.stringify(dhisConfig)
+    })
+];
+
+function log(req, res, opt) {
```

Comment:
log function never used so can be removed. This was only some logging util when using proxy for api calls

Reply:
otherwise LGTM

Reply:
Fixed in 9a4c7ee.